Backpack/Dialog/Classes/BPKDialogView.m

```diff
@@ -192,14 +192,9 @@ - (void)updateIconView {
 
             [self.contentView.topAnchor constraintGreaterThanOrEqualToAnchor:self.iconView.bottomAnchor
                                                                     constant:-(iconViewSize.height / 2.0)],
-
-            // NOTE: The constant here is explicitly not BPKSpacingLg because the icon view has a white border which
```

Comment:
@k0nserv I personally think this comment can go, but want to check with you first.

Reply:
I'd wanna keep it in content where we add the constraint